### DISCUSSION TITLE
Missing clone_count field

### DISCUSSION THREAD

**Discussion by abspangler13** [milaboratory/mixcr#1817] [Q&A]
Hello, I'm running MiXCR4 `analyze` with the smart-seq2 preset. When using MiXCR3 to analyze this same type of data, I used to get a clone_count variable which I believe represented the number of reads for a particular clone in a well. It's missing from the MiXCR4 output now and so, I'm wondering how to get it back or which variable to use instead as a replacement.

**Comment by mizraelson** [comment]
Can you share the column names that you get in the tsv clonotype table? You should have `readCount` column.
